src/main/java/org/symphonyoss/integration/webhook/salesforce/SalesforceWebHookIntegration.java

```diff
@@ -96,4 +109,58 @@ private Entity parsePayloadToEntity(WebHookPayload payload) {
     }
   }
 
-}
+  /**
+   * Return true when Type is JSON, is not tag ContentType when return false
+   * @param payload type WebHookPayload
+   * @return the true when Type is JSON
+   */
+  private boolean isContentTypeJSON(WebHookPayload payload) {
+    if(!hasContentType(payload)) {
+      return false;
+    }
+
+    return MediaType.APPLICATION_JSON.equals(getContentType(payload));
+  }
+
+  /**
+   * Return the type of integration JSON/XML
+   * @param payload type WebHookPayload
+   * @return the Content-Type with Header payload
+   */
+  private String getContentType(WebHookPayload payload) {
+    return payload.getHeaders().get("content-type");
+  }
+
+  /**
+   * Return if exists header ContentType
+   * @param payload type WebHookPayload
+   * @return  if exists the Content-Type with Header payload
+   */
+  private boolean hasContentType(WebHookPayload payload) {
+    return payload.getHeaders().get("content-type") != null;
+  }
+
+  /**
+   * Return the MessageML parser with JSON integration about Opportunity Notification
+   * @param input type WebHookPayload
+   * @return MessageML formatted
+   */
+  private String processIntegrationWithJSON(WebHookPayload input) {
+    JsonNode rootNode = null;
+
+    try {
+      rootNode = JsonUtils.readTree(input.getBody());
+    } catch (IOException e) {
+      throw new SalesforceParseException(
+          "Something went wrong when trying to validate the MessageML received to object.", e);
```

Comment:
Something went wrong when trying parse the JSON payload received by the webhook.